boards/arm/verdin_imx8mp_m7/verdin_imx8mp_m7.dts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2017,2019 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_imx8ml_m7.dtsi>
+#include "verdin_imx8mp_m7-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Toradex Verdin iMX8M Plus M7";
+	compatible = "nxp,mimx8mp_evk";
+
+	aliases {
+		uart-1 = &uart1;
+		uart-4 = &uart4;
+	};
+
+	chosen {
+		/* TCM */
+		zephyr,flash = &itcm;
+		zephyr,sram = &dtcm;
+		/* DDR */
+		/* zephyr,flash = &ddr_code; */
```

Comment:
Something you could consider to enable running from DDR or TCM is to create two different board targets, this is something that has been done for some of the IMX boards: https://github.com/zephyrproject-rtos/zephyr/tree/main/boards/arm/mimx8mp_evk

Reply:
Thanks for the suggestion @danieldegrasse. I have changed the code to add two targets, such as NXP.